pubspec.yaml

```diff
@@ -86,6 +86,11 @@ dependencies:
       url: https://github.com/dab246/super_tag_editor.git
       ref: master
 
+  dns_client:
+    git:
+      url: https://github.com/dab246/dns_client.git
+      ref: twake-supported
```

Comment:
Please create the upstream PR, and add the `todo` in here:
// to do: we will change it when the PR in upstream repository will be merged

Reply:
And link upstream PR?

Reply:
Updated